docker/dockerfile-install-sles

```diff
@@ -0,0 +1,21 @@
+# Parameters related to building rocfft
+ARG base_image
+
+FROM ${base_image}
+LABEL maintainer="saad.rahim@amd"
+
+# Copy the rpm package of rocfft into the container from host
+COPY *.rpm /tmp/
+
+# Install the rpm package, and print out contents of expected changed locations
+RUN zypper -n update && zypper install -n\
+    /tmp/rocfft-*.rpm \
```

Comment:
I'm not sure what the docker install file does, but I know that rocJenkins looks for it. Should change all the instances of rocfft to rocrand